src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1367,4 +1354,38 @@ private void OnSpeciesNameTextChanged(string newText)
             speciesNameEdit.Set("custom_colors/font_color", new Color(1, 1, 1));
         }
     }
+
+    private List<KeyValuePair<string, float>> SortBarData(Dictionary<string, float> bar)
+    {
+        List<KeyValuePair<string, float>> result;
+        ATPComparer atpComparer = new ATPComparer();
+
+        result = bar.OrderBy(
+            i => i.Key, atpComparer)
+            .ToList();
+
+        return result;
+    }
+
+    private class ATPComparer : IComparer<string>
+    {
+        public int Compare(string stringA, string stringB)
+        {
+            bool stringAPriority = stringA == "osmoregulation" || stringA == "baseMovement";
```

Comment:
I think this should also separate osmoregulation to be before baseMovement, as currently the sort is indifferent regarding the order between those two.